n/nlohmann-json/stone.yaml

```diff
@@ -0,0 +1,27 @@
+#
+# SPDX-FileCopyrightText: © 2020-2024 Serpent OS Developers
+#
+# SPDX-License-Identifier: MPL-2.0
+#
+name        : nlohmann-json
+version     : 3.11.3
+release     : 1
+homepage    : https://github.com/nlohmann/json
+upstreams   :
+    - git|https://github.com/nlohmann/json.git : 9cca280a4d0ccf0c08f47a99aa71d1b0e52f8d03
```

Comment:
On Solus we build this with a source archive, is there some reason you need a git checkout? Source archives are strongly preferred when possible